lib/commandsExecute.js

```diff
@@ -35,13 +35,13 @@ var commandsExecute = function(name, params, statusCallback) {
         var timeStamp = Date.now();
         if (res.headers['content-type'] === 'application/json; charset=utf-8') {
             if (res.body.state !== 'inProgress') {
-              deferred.resolve({ body: res.body, response: res.res });
+              deferred.resolve({ response: res.res });
             } else {
               var commandId = res.body.id;
               poll.commandStatus(client, commandId, deferred, timeStamp, statusCallback);
             }
         } else {
-            deferred.resolve({ body: res.body, response: res.res });  // only for getImage command
+            deferred.resolve({ response: res.res });  // only for getImage command
```

Comment:
This probably needs some special handling, last time we tried using this we couldn't even figure out how to get the data at all.